edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code lacks error handling and validation for user inputs.

Recommendations:
- **Input Validation**: Validate inputs to ensure they meet expected formats and constraints before processing.
- **Error Handling**: Implement error handling to manage database connection issues or query failures gracefully.
- **Example**:
  ```php
  if (empty($title) || empty($author) || !is_numeric($year)) {
      echo "Invalid input";
      return;
  }
  ```